python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -493,6 +493,17 @@ def process_graph(args):
     print_graph_info(g, node_data, edge_data)
     sys_tracker.check('Construct DGL graph')
 
+    # reshape customized mask
+    for srctype_etype_dsttype in edge_data:
+        if "train_mask" in edge_data[srctype_etype_dsttype].keys() and \
+            len(edge_data[srctype_etype_dsttype]["train_mask"].shape) == 2:
+            edge_data[srctype_etype_dsttype]["train_mask"] = \
+                edge_data[srctype_etype_dsttype]["train_mask"].squeeze(1).astype('int8')
+            edge_data[srctype_etype_dsttype]["val_mask"] = \
```

Comment:
You need to check train_mask, val_mask and test_mask separately.

Reply:
addressed